compiler-rt/test/tsan/simple_stack_symbolizer_markup.cpp

```diff
@@ -0,0 +1,57 @@
+// REQUIRES: linux
+// RUN: %clangxx_tsan %s -Wl,--build-id=0x12345678 -O1 -o %t.main
+// RUN: mkdir -p %t/.build-id/12
+// RUN: cp %t.main %t/.build-id/12/345678.debug
+// RUN: %env_tsan_opts=enable_symbolizer_markup=1 %deflake %run %t.main >%t/sanitizer.out
+// RUN: llvm-symbolizer --filter-markup --debug-file-directory=%t < %t/sanitizer.out | FileCheck %s --dump-input=always
+
+#include "test.h"
+
+int Global;
+
+void __attribute__((noinline)) foo1() { Global = 42; }
```

Comment:
why mix `[[gnu::noinline]]` and `__attribute__((noinline))`? I expect these are copied from existing test files, but I think you should be consistent in this patch. I think we prefer cpp style attributes when possible.

Reply:
Yes, I copied another test and modified it. My assumption was that it would be better to keep each test as an analogous to the original, so I tried to keep it as close to the original as possible. but I totally agree with your comment.